features/nvstore/source/nvstore.cpp

```diff
@@ -597,16 +599,16 @@ int NVStore::do_get(uint16_t key, uint16_t buf_size, void *buf, uint16_t &actual
     }
 
     _mutex->lock();
-    record_offset = _offset_by_key[key];
+	
+	record_offset = _offset_by_key[key];
```

Comment:
misaligned- tabs here?

Reply:
Strange, my editor is setup to insert spaces only. Anyway, fixed it.